pkg/valueset/postgresql.go

```diff
@@ -0,0 +1,58 @@
+package pkg
+
+import (
+	"io/ioutil"
+	"net/url"
+
+	"gopkg.in/yaml.v2"
+)
+
+const hostName = "localhost"
+const serverPort = 8011
+
+type Config struct {
```

Comment:
```
Config {
   []Database {
       creds and URL
       []Valuesets
}
```
add a config.getConnection(valueset) -> connect if necessary
Connect once, keep in config.
Have a config.Close() to close all open connections